Sun Apr 15 14:11:13 FET 2012  Sergei Trofimovich <slyfox@community.haskell.org>
  * .cabal: allow transformers-0.3
diff -rN -u old-hipmunk/Hipmunk.cabal new-hipmunk/Hipmunk.cabal
--- old-hipmunk/Hipmunk.cabal	2012-04-15 14:12:26.373740233 +0300
+++ new-hipmunk/Hipmunk.cabal	2012-04-15 14:12:26.448739297 +0300
@@ -195,7 +195,7 @@
     Build-Depends: base >= 3 && < 5,
                    array >= 0.1 && < 0.5,
                    containers >= 0.1 && < 0.5,
-                   transformers >= 0.2 && < 0.3,
+                   transformers >= 0.2 && < 0.4,
                    StateVar >= 1.0 && < 1.1
   else
     Build-Depends: base >= 2 && < 3,
